proposals/0008-swift-prometheus.md

```diff
@@ -0,0 +1,150 @@
+# SwiftPrometheus - Prometheus Metrics in Swift
+
+* Proposal: SSWG-0008
+* Authors: [Jari Koopman / LotU](https://github.com/MrLotU)
+* Review Manager: TBD
+* Status:  [Implemented ](https://github.com/MrLotU/SwiftPrometheus/tree/MetricsLib)
+* Pitch: [Pitches/Prometheus](https://forums.swift.org/t/client-side-prometheus-implementation/18098/)
+
+## Package Description
+Prometheus client side implementation.
+
+|  |  |
+|--|--|
+| **Package Name** | `SwiftPrometheus` |
```

Comment:
i suhhest SwiftPrometheusClient or something along these lines since this is a client library to a prometheus server

Reply:
What I've seen from Prometheus clients in other languages is that they generally don't include `Client` in the name, since most (if not all) packages are clients. The server side is provided by Prometheus itself and there's not much of a point porting it to another language.

Reply:
/cc @swift-server/sswg i believe we decided we wanted to have names that are explicit

Reply:
i can see why we could look the other way in this case as its pretty clear this is not "prometheus" the server, but /cc @weissi  and rest of @swift-server/sswg  in case they feel strongly about it

Reply:
I don't feel strongly - happy either way.